lib/netbox_client_ruby/api/dcim.rb

```diff
@@ -6,6 +6,8 @@
 require 'netbox_client_ruby/api/dcim/device_types'
 require 'netbox_client_ruby/api/dcim/interface'
 require 'netbox_client_ruby/api/dcim/interfaces'
+require 'netbox_client_ruby/api/dcim/inventory_item'
```

Comment:
I would only require the "proxy file" in here, namely `inventory_items`, and you can require `inventory_item` in that proxy file.

That can also be applied to the other requires you have in here.